spacer_floating_platform: #namespace
  fp_valve_controller: #node name
    ros__parameters:
      device: 'rpi'
      topic_name: 'valves/input'
      gpio_mode: 'BCM'
      gpio_ids: [15, 7, 24, 26, 31, 29, 23, 21, 19] #(B9, T1, T2, ..., T8)